docker/misk-web/misk-web

```diff
@@ -0,0 +1,127 @@
+#!/bin/sh
```

Comment:
Bash city! I hear Kotlin can be used for scripting? I don't suggest re-writing this or anything now that it's done, but maybe consider using that next time for your own sanity ;)

Reply:
💯 bash is a nightmare of wonder